### DISCUSSION TITLE
Github Sponsorships for open source codes - How to get noticed?

### DISCUSSION THREAD

**Discussion by MarinJurcic20** [EddieHubCommunity/support#568] [Q&A] [answered]
Hello to all,

I'm writing to ask if any of you has any experience of getting sponsored on Github - to be exact how do you attract sponsors when your open source codes are not meant for a super commercial niche (a open sheet music player w. audio)? This isn't a standard Patreon thing nor a crowdfunding campaign and so it's kinda like sailing in uncharted waters so any advice is more than welcome.

I'm new to this community,  I'm not a programmer, but a digital marketer (some of my closest friends are coders and open source enthusiasts) and I figured this is the best place to ask. 

Kind regards to this community,

Marin

**Comment by eddiejaoude** [comment, marked as answer]
Hi Marin,

Thank you for asking the question. I use GitHub Sponsors, I have almost 10 sponsors and I sponsor about 5 people.

My suggestions are:

- Have a complete and activity GitHub profile
- Add value to the open source community with open source project (these do not just have to be code) and also add value to other projects on GitHub (again does not have to be code)
- Document the journey of the projects (share the content transparently on other social platforms)

As you get going, more avenues will become apparent.

I hope that helps 🤓

**Reply by MarinJurcic20** [reply]
Hey Eddie,

thanks for the tips & suggestions. We're already active on GH, currently finishing up a project - it's about displaying sheet music. You can have a look at our repositories here:  https://github.com/opensheetmusicdisplay

As I'm a digital marketer and not a dev (didn't mean to be sneaky but rather ask for advice). I haven't found a lot of case studies or guides on building up an sponsorship audience on GH, I'll try to document this process and share my experiences with other devs so it might come in handy. 

We kicked off our sponsorship yesterday, if anyone feels generous or would like to check out the perks - here's the link:  https://github.com/sponsors/opensheetmusicdisplay

Thank you a lot :)

**Reply by eddiejaoude** [reply]
Yes definitely document and share your journey 👍 

Also I recommend to have all your team members fill out their profile, it will look more professional and inclusive...

<img width="925" alt="Screenshot 2020-12-28 at 10 07 44" src="https://user-images.githubusercontent.com/624760/103207196-a5c0de80-48f5-11eb-9b8d-d6c129221171.png">

**Comment by MarinJurcic20** [comment]
We've let them know, just to let you know we got our first sponsor.

Merry Christmas Eddie and best wishes for 2021. from PhonicScore/OSMD :)

**Reply by eddiejaoude** [reply]
Awesome on your first sponsor!! 🎉
